deploy_flask.sh

```diff
@@ -8,7 +8,11 @@ apt install -y python3-pip
 pip3 install --upgrade pip
 pip3 install Flask 
 
-#go to the directory containing the file we want to run
+apt install -y postgresql postgresql-contrib
+
 cd /vagrant/
 
-FLASK_APP=hello.py flask run --host=0.0.0.0
+FLASK_APP=hello.py
+sudo python3 hello.py >> log.txt 2>&1 &
```

Comment:
why do you need `sudo`?

Reply:
I don't. It was already a part of the deploy_flask.sh, I'll check if the script works without it and remove the sudo if it does.

Reply:
@igoihman removed.